Adjust comment styling for extended cells to ensure consistent appearance

banobepascal opened the pull request:
Change-Id: Ia3b6ac17ad1f4677e61d9128806bac90a819bfb3


* Resolves: #10762 
* Target version: master 

### Summary
This fix addresses an issue where comments in cells appeared misaligned or visually inconsistent when the cell was extended

### Visual Representation
![Screenshot from 2024-12-25 11-43-11](https://github.com/user-attachments/assets/3d56d731-6db8-4589-8d0f-8c684a2018cf)
![Screenshot from 2024-12-25 11-43-37](https://github.com/user-attachments/assets/47d2c01c-a8d6-4cfa-9d68-1f1029ba15af)



### Checklist

- [x] I have run `make prettier-write` and formatted the code.
- [x] All commits have Change-Id
- [x] I have run tests with `make check`
- [x] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

banobepascal commented:
@eszkadev please review

eszkadev reviewed (changes requested):
I think it requires fix for RTL sheet `layout->RTL` (not confuse with RTL UI `<URL>&lang=ar`)

![rtl](https://github.com/user-attachments/assets/5fdd5f09-5544-4e0e-85d2-e75f29060003)

Also it is now not aligned to the cell border (has some gap), is it intended?

banobepascal commented:
> I think it requires fix for RTL sheet `layout->RTL` (not confuse with RTL UI `<URL>&lang=ar`)
> 
> ![rtl](https://private-user-images.githubusercontent.com/5307369/397739857-5fdd5f09-5544-4e0e-85d2-e75f29060003.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUxMTYzNjUsIm5iZiI6MTczNTExNjA2NSwicGF0aCI6Ii81MzA3MzY5LzM5NzczOTg1Ny01ZmRkNWYwOS01NTQ0LTRlMGUtODVkMi1lNzVmMjkwNjAwMDMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIyNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMjVUMDg0MTA1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzM4ZGFiNjE2OTc0NzcxNTJjNjdiOGMzMDZhYmY2ODlmYTM5NTgxNmU2NDVkODAyNDQ3YWU0ZDFjNmNmYjRlMyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.f2V1E5Ht3Cfb2rOTGgmsIsPqYaeqyV-RLJw3Rak0lwM)
> 
> Also it is now not aligned to the cell border (has some gap), is it intended?

Removed gap and also fixed RTL 

![Screenshot from 2024-12-25 11-43-37](https://github.com/user-attachments/assets/874b06c9-d783-4473-9df6-1505dbc4fe85)
![Screenshot from 2024-12-25 11-43-11](https://github.com/user-attachments/assets/4648e289-ba8f-4732-9e7c-78e43cd134fa)

banobepascal commented:
@eszkadev please review the changes

eszkadev commented:
I can confirm now it is better and RTL works.
I found that with lower zoom we still have the same size of triangle as with large zoom.
So with 50% it looks bigger than a cell and with 200% small:
![zoom](https://github.com/user-attachments/assets/1f838012-e87d-488b-8976-21675362f1e0)

Also in RTL there is 1px gap on the left edge if we want to be pixel-perfect.

banobepascal commented:
> I can confirm now it is better and RTL works. I found that with lower zoom we still have the same size of triangle as with large zoom. So with 50% it looks bigger than a cell and with 200% small: ![zoom](https://private-user-images.githubusercontent.com/5307369/398700783-1f838012-e87d-488b-8976-21675362f1e0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUyMjM2MDIsIm5iZiI6MTczNTIyMzMwMiwicGF0aCI6Ii81MzA3MzY5LzM5ODcwMDc4My0xZjgzODAxMi1lODdkLTQ4OGItODk3Ni0yMTY3NTM2MmYxZTAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIyNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMjZUMTQyODIyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTJiMzQ5MjUxYjcyMmY5OGVlZTBmZjg0ODZiYzJlM2QwNjQ4NjRjOGZmOGUwMDQ4MmY0NzhiNWRlY2U0ZmY0ZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.F34dHRW2Cn2EWHc_Xy5Hth3FwwHBLWJZ4JFj6-zdz4k)
> 
> Also in RTL there is 1px gap on the left edge if we want to be pixel-perfect.

The triangle is now adjusting according to the zoom level
![Screenshot from 2024-12-26 17-24-53](https://github.com/user-attachments/assets/b056c2f8-2d1f-43a1-a8b3-8dc67fae0c7b)
![Screenshot from 2024-12-26 17-24-23](https://github.com/user-attachments/assets/ae6ec939-97ea-4331-8b07-6a7f3f63d143)


The reason for the RTL margin 1px is because of the column borders are not straight with the top bar so if the column or row borders are adjusted this creates a misalignment.
![Screenshot from 2024-12-26 17-22-20](https://github.com/user-attachments/assets/9b0600f1-6aec-44d5-bfba-54dede575917)

eszkadev reviewed (approved):
works now :)
